src/Engine/ProtoCore/Lang/FFIFunctionEndPoint.cs

```diff
@@ -84,9 +84,7 @@ private void Init(RuntimeCore runtimeCore)
         }
 
         public override StackValue Execute(Runtime.Context c, List<StackValue> formalParameters, StackFrame stackFrame, RuntimeCore runtimeCore)
-        {   
-            StackValue svThisPtr = stackFrame.ThisPtr;
```

Comment:
Was the stackFrame null in some cases? If so, was this crashing before, is this why you added the null check in line 98?

Reply:
This check is new.  The `stackFrame` is not passed here when being invoked from the `DisposeDispatch`.  This is now a check for that case so it skips the processing necessary for a generic call and jumps to Execute.